pkg/connector/slack/v0/config/definition.json

```diff
@@ -15,7 +15,7 @@
       "additionalProperties": false,
       "properties": {
         "token": {
-          "description": "Fill your token",
+          "description": "Please go to the Slack API tutorial to create a Slack App, set the scope of token, and get the token.",
```

Comment:
- `description` should _primarily_ describe the field, so I'd include the (valuable) information about how to create one as a 2nd sentence. 
- I feel `documentation` is a better choice than `tutorial`. The source of truth should be the former, not the latter.
- I added a suggestion to align with the existing connector credential fields.

```suggestion
          "description": "Fill in your Slack app access token. For more information about how to access o create app tokens, please refer to the Slack API documentation.",
```

Reply:
@jvallesm 
Thanks for your clear suggestion. I will modify it with your comment.

There is a small point about `documentation` vs. `tutorial` that I want to know your opinion.

When I integrated the Slack API, I felt the information in `[Guided tutorials](https://api.slack.com/tutorials)` is easier for me to catch up than [Web API methods](https://api.slack.com/methods) or other links in the search in the first image.

Note
- But, it could be also related to SEO and it will be changed in the future in Google Search.

What do you think about it?

![image](https://github.com/instill-ai/component/assets/64747455/1f8228ab-a5c7-42e7-9bfd-b99f23cdd617)
![image](https://github.com/instill-ai/component/assets/64747455/8b82fca7-81b7-48db-98d3-a1ebfb6fa4a2)

Reply:
Definitely [Web API methods](https://api.slack.com/methods) is too broad. I wasn't familiar myself with slack apps but I searched "slack apps", where that page isn't among the first results.

![CleanShot 2024-05-21 at 13 28 49](https://github.com/instill-ai/component/assets/3977183/ff33ddaa-a12b-4f7a-8004-3087a9fc74f8)
![CleanShot 2024-05-21 at 13 28 13](https://github.com/instill-ai/component/assets/3977183/571e0da7-81c8-491e-928b-286a4adbffdf)

I think it's indeed a SEO issue and that, conceptually, we should refer to the documentation rather than the tutorial. Think that a user might have a slack app already created, or do so as the first step in the Instill AI integration, so pointing them to the tutorial isn't as relevant as to the API documentation.

Reply:
OKOK, thank you! I will modify them and push again later!